rcljava/src/main/cpp/org_ros2_rcljava_RCLJava.cpp

```diff
@@ -58,20 +97,46 @@ Java_org_ros2_rcljava_RCLJava_nativeCreateNodeHandle(
 
   rcl_context_t * context = reinterpret_cast<rcl_context_t *>(context_handle);
 
+  rcl_arguments_t arguments;
+  if (!parse_arguments(env, cli_args, &arguments)) {
+    // All of the exception setup was done by parse_arguments, just return here.
+    return 0;
+  }
+
   rcl_node_t * node = static_cast<rcl_node_t *>(malloc(sizeof(rcl_node_t)));
```

Comment:
If the suggestion above is ignored, please check the value returned by `malloc`.

Reply:
Should be fixed in ad331e5.